packages/react-bindings/src/styles/types.ts

```diff
@@ -68,18 +68,24 @@ export type Renderer = Omit<FelaRenderer, 'renderRule'> & {
 }
 
 export interface StylesContextPerformance {
-  enableStylesCaching?: boolean
-  enableVariablesCaching?: boolean
+  enableSanitizeCssPlugin: boolean
+  enableStylesCaching: boolean
+  enableVariablesCaching: boolean
 }
 
+export type StylesContextPerformanceInput = Partial<StylesContextPerformance>
+
 export type StylesContextInputValue<R = Renderer> = {
   disableAnimations?: boolean
-  performance?: StylesContextPerformance
+  performance?: StylesContextPerformanceInput
   renderer?: R
   theme?: ThemeInput
 }
 
-export type StylesContextValue<R = Renderer> = Required<StylesContextInputValue<R>> & {
+export type StylesContextValue<R = Renderer> = {
+  disableAnimations: boolean
+  performance: StylesContextPerformanceInput
+  renderer: R
```

Comment:
Like in other places these things are clearly separated: `Input` can be optional while the result of merge will contain all values.